IS0501Test.py

```diff
@@ -55,8 +55,8 @@ def test_01(self, test):
         dest = ""
         valid, result = self.is05_utils.checkCleanRequestJSON("GET", dest)
         if valid:
-            msg = "Got the wrong json from {} - got {}. Please check json matches the spec, including trailing slashes" \
-                .format(dest, result)
+            msg = "Got the wrong json from {} - got {}. Please check json matches the spec, including trailing " \
+                  "slashes".format(dest, result)

```

Comment:
Is the line-continuation '\' needed in this and the next one? I haven't got my head around why it sometimes is, and why it sometimes isn't.

Reply:
If the statement is fully enclosed by brackets (such as an argument to a method) you don't need the continuation '\'. In this case it either requires the '\', or enclosing the string in brackets.

Reply:
Oh! Thanks. Makes sense.